src/pages/UsersIndex/UsersIndex.tsx

```diff
@@ -70,24 +70,20 @@ export const UsersIndex = () => {
   const [errorFetch, setErrorFetch] = useState<AxiosError>()
   const [users, setUsers] = useState<SyncUser[]>([])
   const { getUsers } = useUsers()
-
-  useEffect(() => {
-    updateUsers()
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [])
+  const isMounted = useRef<boolean>(false)
 
   /**
    * Fetch fresh list of users and set to useState
    */
-  const updateUsers = () => {
+  const updateUsers = useCallback(() => {
```

Comment:
This refactor is part of an effort to eliminate/reduce warnings from unit tests for Routes.tsx.